blocks/init/src/Blocks/components/button/button.php

```diff
@@ -71,7 +71,7 @@ class="<?php echo esc_attr($buttonClass); ?>"
 	<?php
 	foreach ($buttonAttrs as $key => $value) {
 		if (!empty($key) && !empty($value)) {
-			echo wp_kses_post("{$key}=" . $value . " ");
+			echo wp_kses_post("{$key}=\"$value\"");
```

Comment:
Excuse me. This 👆🏻 is a mess->mess fix 🙃
```suggestion
			echo wp_kses_post(sprintf('%s="%s"', $key, $value));
```

Reply:
Yeah, I don't think there are benefits of using a function call, where the same can be achieved using simple string interpolation...

Reply:
I think you have not read my policy on string manipulation: https://github.com/szepeviktor/very-clean-code/blob/master/README.md#-string-manipulation

But I let you go in this direction.